download_and_create_reference_datasets/v03/write_hgmd_ht.py

```diff
@@ -2,43 +2,23 @@
 import argparse
 import os
 
-from hail_scripts.reference_data.combine import (
-    get_enum_select_fields,
-    get_select_fields,
-    update_joined_ht_globals,
-)
-from hail_scripts.reference_data.config import CONFIG, GCS_PREFIXES, AccessControl
-from hail_scripts.utils.hail_utils import import_vcf, write_ht
+from hail_scripts.reference_data.combine import get_ht
+from hail_scripts.reference_data.config import GCS_PREFIXES, AccessControl
+from hail_scripts.utils.hail_utils import write_ht
 
-DATASETS = ['hgmd']
 HGMD_HT_PATH = 'hgmd/hgmd.GRCh{genome_version}.ht'
 PARTITIONS = 100
-VERSION = '1.0.0'
 
 
 def run(environment: str, genome_version: str):
     dataset = 'hgmd'
-    source_path = CONFIG[dataset][genome_version]['path']
     destination_path = os.path.join(
         GCS_PREFIXES[(environment, AccessControl.PRIVATE)],
         HGMD_HT_PATH,
     ).format(
         genome_version=genome_version,
     )
-    mt = import_vcf(
```

Comment:
I went ahead and ported `hgmd` into the `config.py` framework so we can just call `get_ht` here rather than a subset of the functions that `get_ht` calls.